pkg/workspace/library_module.go

```diff
@@ -318,7 +317,8 @@ func (l *libraryValue) libraryValues(ll *LibraryLoader) (*DataValues, []*DataVal
 		}
 	}
 
-	dvs, foundChildDVss, err := ll.Values(append(dvss, afterLibModDVss...), &schema.AnySchema{})
+	schema, err := ll.Schemas()
+	dvs, foundChildDVss, err := ll.Values(append(dvss, afterLibModDVss...), schema)
```

Comment:
it seems that we are using ll.Schemas() and then passing it to ll.Values(). i would argue Values() shouldnt require schema as input and should just use ll.Schemas() internally.